packages/mongodb-rag-core/src/test/MockPageStore.ts

```diff
@@ -9,5 +9,8 @@ export const makeMockPageStore = (): PageStore => {
     async updatePages(args: PersistedPage[]) {
       pages = [...args];
     },
+    async deletePages(_args: DeletePagesArgs) {
```

Comment:
```suggestion
    async deletePages(_args) {
```

pretty sure you dont need this b/c it comes from the type

this lets you delete the imprted type as well